nav2_behavior_tree/CMakeLists.txt

```diff
@@ -111,6 +111,9 @@ list(APPEND plugin_libs nav2_truncate_path_action_bt_node)
 add_library(nav2_goal_updater_node_bt_node SHARED plugins/decorator/goal_updater_node.cpp)
```

Comment:
See this PR, this needs to be added to all of the param files / default BT plugin lists https://github.com/ros-planning/navigation2/pull/2236/files

Reply:
Isnt this requirement completed in this second commit?